python/tests/conftest.py

```diff
@@ -25,12 +25,9 @@
 import attr
 import pytest
 import simplejson
-from _pytest.config import Config
-from _pytest.fixtures import FixtureRequest
```

Comment:
maybe this voodoo was required in earlier pytest versions but it is no longer the case